packages/firestore/test/unit/specs/describe_spec.ts

```diff
@@ -24,6 +24,8 @@ import { addEqualityMatcher } from '../../util/equality_matcher';
 import { SpecBuilder } from './spec_builder';
 import { SpecStep } from './spec_test_runner';
 
+import * as stringify from 'json-stable-stringify';
```

Comment:
It's less important in tests but we should be getting in the habit of avoiding whole-module imports like this.

However, I'm not actually sure what the right way to do this import is. Does `import stringify from 'json-stable-stringify'` work?

Reply:
Ah, it looks like we're missing the `esModuleInterop` compiler setting.

@Feiyang1 is there any reason not to enable this repo-wide? We're already using it in our [integration tests](https://github.com/firebase/firebase-js-sdk/blob/6668b59e2fa495fcea42a24a6b24821407ee703f/integration/typescript/tsconfig.json#L13).

Reply:
We can enable it. I'm actually surprised that you can use `stringify` as a function directly. Thought typescript will give an error as it's not compliant with es6 modules spec.

Reply:
https://github.com/firebase/firebase-js-sdk/pull/2849